pulseapi/entries/views.py

```diff
@@ -117,6 +117,17 @@ class EntryView(RetrieveAPIView):
     pagination_class = None
 
 
+class FavoriteEntries(ListAPIView):
```

Comment:
Hmm, could we implement this as, itself, a filter? I.e. when users call `/entries/favorites[?args]` Django internally serves up the result for `/entries/[?args]&is_favorited=True`?